### ISSUE TITLE
auto-friendships and auto-group joins

### ISSUE THREAD

**Issue by boonebgorges** [boonebgorges/invite-anyone#6]
Request: When an invited person joins the site, instead of sending invitations to groups and friendships, make them automatic. Do we want a toggle for this? In the UI or as a constant?

**Comment by Jon007** [comment]
Yes, this would be great, and it should be a toggle defaulting to off to avoid impacting existing users :)
